doc/configuration.md

```diff
@@ -139,6 +140,16 @@ tags:
 This field is not supported in the
 [global configuration file](#global-configuration).
 
+### `retry`
+
+This int field controls how many times a test is retried upon failure. 
+
+```yaml
+tags:
+  chrome:
+    retry: 3 # retry chrome failures 3 times
+```
+
```

Comment:
Global config is intended to allow users to configure things that may be system-wide—for example, their computer may be particularly slow, so they could set `timeout: 1.5x` globally. Or they may personally prefer verbose stack traces, so they could set `verbose_trace: true` globally.

But we don't want to let their global configuration make tests run differently on their system; that would be a recipe for painful bugs. The line between those two is subjective, but broadly we try to only allow global configuration that has to do with their system as a whole or their personal preferences.